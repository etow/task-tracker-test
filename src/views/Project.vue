<template>
  <Board />
  <EditTask />
</template>
<script>
import Board from '../components/Board/Board.vue';
import EditTask from '../components/EditTask/EditTask.vue';

export default {
  components: {
    Board,
    EditTask,
  },
}
</script>
