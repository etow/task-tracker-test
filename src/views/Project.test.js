import { shallowMount } from '@vue/test-utils'
import Project from './Project.vue'

describe('Project.vue', () => {
  it('It renders correctly', () => {
    const wrapper = shallowMount(Project)
    expect(wrapper.html()).toMatchSnapshot();
  })
})
