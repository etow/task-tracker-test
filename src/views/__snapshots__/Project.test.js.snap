// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project.vue It renders correctly 1`] = `
<board-stub></board-stub>
<edit-task-stub></edit-task-stub>
`;
